etc/kayobe/ansible/octavia-amphora-image-build.yml

```diff
@@ -4,6 +4,12 @@
   vars:
     amphora_dib_upper_constraints_file: "{{ pip_upper_constraints_file }}"
   tasks:
+    - name: Install EPEL
+      package:
+        name: epel-release
```

Comment:
We generally try to avoid enabling EPEL. But if we need it, we might want to consider temporarily enabling it. Do we need to consider using the release train EPEL repo here?

Reply:
So this is for debootstrap on Rocky. At least in the environment I was testing, the seed wasn't configured to use release train snapshots, but fair enough, the builder group is customisable. We could also consider building Rocky based images on a Rocky Host OS?

Reply:
https://github.com/stackhpc/stackhpc-kayobe-config/blob/699769ce3756c6dc80c1183e1c87f730ff05bb80/etc/kayobe/inventory/group_vars/overcloud/stackhpc-repos#L5

Reply:
That's true - in general it doesn't use release train repos.